apps/ledger-live-mobile/src/components/ReadOnlyAccountGraphCard.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { memo } from "react";
+import { useTheme } from "styled-components/native";
+import { useSelector } from "react-redux";
+import { Unit, CryptoCurrency } from "@ledgerhq/live-common/lib/types";
+import { ValueChange } from "@ledgerhq/live-common/lib/portfolio/v2/types";
+import { Flex, Text } from "@ledgerhq/native-ui";
+
+import Delta from "./Delta";
+import getWindowDimensions from "../logic/getWindowDimensions";
+import Graph from "../icons/Graph";
+import { withDiscreetMode } from "../context/DiscreetModeContext";
+import DiscreetModeButton from "./DiscreetModeButton";
+import CurrencyUnitValue from "./CurrencyUnitValue";
+import { counterValueCurrencySelector } from "../reducers/settings";
+
+const { width } = getWindowDimensions();
+
+type HeaderTitleProps = {
+  valueChange: ValueChange;
+  cryptoCurrencyUnit: Unit;
+};
+
+const GraphCardHeader = ({
```

Comment:
It's really really similar to Come's GraphCard at src/screens/Portfolio/ReadOnly/ReadOnlyGraphCard.tsx !
Something could be done to reduce duplicated code